core/src/main/java/com/gentics/mesh/core/endpoint/node/BinaryTransformHandler.java

```diff
@@ -226,11 +226,12 @@ public void handleS3TransformImage(RoutingContext rc, String uuid, String fieldN
 				.handleS3Resize(options.getS3Options().getBucket(), s3ObjectKey, fileName, parameters)
 				.flatMap(file -> {
 					// The image was stored and hashed. Now we need to load the stored file again and check the image properties
-					Single<ImageInfo> info = imageManipulator.readImageInfo(file.getName());
-					Single<FileProps> fileProps = fs.rxProps(file.getName());
+					String filePath = file.getAbsolutePath();
```

Comment:
The absolute path still stays in Mesh, and is needed for the further steps of an async processing to pick the cached image from another project scope.